public/addEdit.js

```diff
@@ -0,0 +1,144 @@
+import { enableInput, inputEnabled, message, setDiv, token } from "./index.js";
+import { showTires } from "./tires.js";
+
+
+let addEditDiv = null;
+let brand = null;
+let size = null;
+let location = null;
+
+// if left uncommented it breaks the addingTire option
+// let price = null;
+// let quantity = null;
+let addingTire = null;
+
+
+export const handleAddEdit = () => {
+    addEditDiv = document.getElementById("edit-tire");
+    brand = document.getElementById("brand");
+    size = document.getElementById("size");
+    location = document.getElementById("location");
+    // price = document.getElementById("price");
```

Comment:
What exact error are you seeing when these are uncommented? My guess is you’ll need to convert since the mongoose schema expects a number but DOM element retrieval can be…interesting for typing